src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+/**
+ * Represents a todo task that can store the description and status of the task.
+ */
+public class Todo extends Task {
+    /**
+     * Initializes a Todo with the description and status of not done.
+     *
+     * @param description the description of the task
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String getTaskType() {
+        return "T";
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
```

Comment:
perhaps you could make use of getTaskType() method to return toString(), e.g. `return "[" + getTaskTypeSymbol() + "]" + super.toString();`. Other than this, good job on ensuring coding standards throughout your code!